app/deepseg.cc

```diff
@@ -268,15 +326,15 @@ int main(int argc, char* argv[]) try {
 	size_t threads= 2;
 	size_t width  = 640;
 	size_t height = 480;
-	const char *back = nullptr; // "images/background.png";
-	const char *vcam = "/dev/video0";
-	const char *ccam = "/dev/video1";
+	const char *back = nullptr;
+	const char *vcam = "/dev/video1";
+	const char *ccam = "/dev/video0";
```

Comment:
We might want to be more intelligent in defaulting camera paths too (ie: search for them)?

Reply:
You mean like a path resolution that accepts `video1` in place of `/dev/video1`? I.e. transparently do path lookup using `/dev` for filename-only names?

Reply:
I was thinking of assuming the first video device is the physical camera (typically `/dev/video0`), and the last is the virtual camera, using a similar scanning technique to OBS (which enumerates `/sys/class/video4linux`)

Reply:
Yes. Best split this into another change. But assuming that without a path this is meant to reside in `/dev` might be worthwhile in the meantime.